app/i18n/locales/ar.js

```diff
@@ -0,0 +1,528 @@
+	'error-invalid-password': 'رمز مرور خاطئ',
+	'error-invalid-redirectUri': 'redirectUri غير صحيح',
+	'error-invalid-role': 'وظيفة غير صالح',
+	'error-invalid-room': 'غرفة غير صالحة',
+	'error-invalid-room-name': '{{room_name}} ليس اسم الغرفة صالح',
+	'error-invalid-room-type': '{{type}} ليس نوع الغرفة صالحا.',
+	'error-invalid-settings': 'الإعدادات المعطاة غير صالحة',
+	'error-invalid-subscription': 'اشتراك غير صالح',
+	'error-invalid-token': 'الرمز غير صالح',
+	'error-invalid-triggerWords': 'triggerWords  غير صالح',
+	'error-invalid-urls': 'عناوين  غير صالحة',
+	'error-invalid-user': 'مستخدم  غير صالح',
+	'error-invalid-username': 'إسم المستخدم  غير صالح',
+	'error-invalid-webhook-response': 'الرد التلقائي من العنوان أستجاب برمز مغاير عن المفترض 200',
+	'error-message-deleting-blocked': 'حذف الرسالة محظور',
+	'error-message-editing-blocked': 'تعديل الرسالة محظور',
+	'error-message-size-exceeded': 'حجم الرسالة تجاوز المحدد الحجم المحدد له (Message_MaxAllowedSize)',
+	'error-missing-unsubscribe-link': 'يجب عليك تقديم رابط [unsubscribe]',
+	'error-no-tokens-for-this-user': 'لا توجد رموز مميزة لهذا المستخدم',
+	'error-not-allowed': 'غير مسموح',
+	'error-not-authorized': 'غير مصرح',
+	'error-push-disabled': 'الدفع معطل',
+	'error-remove-last-owner': 'هذا هو المالك الأخير. يرجى تعيين مالك جديد قبل إزالة هذا المالك.',
+	'error-role-in-use': 'لا يمكن حذف الدور لأنه قيد الاستخدام',
+	'error-role-name-required': 'اسم الدور مطلوب',
+	'error-the-field-is-required': 'هذا الحقل {{field}} مطلوب.',
+	'error-too-many-requests': 'خطأ ، الكثير من الطلبات. من فضلك أبطء السرعة. يجب الانتظار لمدة {{seconds}} ثانية قبل المحاولة مرة أخرى.',
+	'error-user-is-not-activated': 'المستخدم غير منشط',
+	'error-user-has-no-roles': 'يس للمستخدم أدوار',
+	'error-user-limit-exceeded': 'يتجاوز عدد المستخدمين الذين تحاول دعوتهم إلى #channel_name الحد الذي حدده المشرف',
+	'error-user-not-in-room': 'المستخدم ليس في هذه الغرفة',
+	'error-user-registration-custom-field': 'error-user-registration-custom-field',
+	'error-user-registration-disabled': 'تم تعطيل تسجيل المستخدم',
+	'error-user-registration-secret': 'تسجيل المستخدم مسموح به فقط عبر عنوان الويب السري',
+	'error-you-are-last-owner': 'أنت المالك الأخير. يرجى تعيين مالك جديد قبل مغادرة الغرفة.',
+	Actions: 'أجراءات',
+	activity: 'نشاط',
+	Activity: 'نشاط',
+	Add_Reaction: 'إضافة تفاعل',
+	Add_Server: 'إضافة خادم',
+	Add_users: 'إضافة مستخدمين',
+	Admin_Panel: 'لوحة الادارة',
+	Alert: 'إنذار',
+	alert: 'إنذار',
+	alerts: 'إنذارات',
+	All_users_in_the_channel_can_write_new_messages: 'يمكن لجميع المستخدمين في القناة كتابة رسائل جديدة',
+	All: 'الكل',
+	All_Messages: 'كل الرسائل',
+	Allow_Reactions: 'السماح للتفاعلات',
+	Alphabetical: 'أبجدي',
+	and_more: 'وأكثر',
+	and: 'و',
+	announcement: 'إعلان',
+	Announcement: 'إعلان',
+	Apply_Your_Certificate: 'تطبيق شهادتك',
+	Applying_a_theme_will_change_how_the_app_looks: 'سيؤدي تطبيق السمة إلى تغيير شكل التطبيق.',
+	ARCHIVE: 'أرشيف',
+	archive: 'أرشيف',
```

Comment:
is this used for the action "archive"? if so it then it should be: 
```suggestion
	archive: 'أرشفة',
```